src/applications/vaos/tests/e2e/flows/appointment-list-flow/upcoming-appointments.cypress.spec.js

```diff
@@ -29,6 +29,7 @@ describe('VAOS upcomming appointment flow', () => {
       mockFeatureToggles();
       mockLoginApi();
       mockVamcEhrApi();
+      mockAppointmentsApi();
```

Comment:
This should be here since the response is different for each test. There are additional changes coming that remove the default of just returning whatever appointments are defined in confirmed.json. The goal is to be more deterministic about what the helper functions expect and return.

Reply:
Thanks for explaining. Will remove this line.